src/containers/auth/login/index.js

```diff
@@ -17,6 +25,19 @@ class Login extends Component {
     errors: {},
   };
 
+  componentDidMount() {
+    const { location, history } = this.props;
```

Comment:
if you destructed location and history, why did you not destruct googleLoginRequest and linkedInLoginRequest from props?